locales/pt_BR/LC_MESSAGES/duckduckgo.po

```diff
@@ -4770,6 +4762,8 @@ msgid ""
 "You're now searching privately in Chrome. Use our app instead of Chrome to "
 "browse privately too. It’s already installed and can:"
 msgstr ""
+"Você está navegando anonimamente no Chrome. Usa nosso aplicativo no lugar do "
+"Chrome para navegar anonimamente também. Já está instalado e pode:"
```

Comment:
@marcellohalfeld Thanks for your help translating these strings. 🙇‍♂️ Here we want to highlight the difference between _searching_ privately and _browsing_ privately (when using the app). With "Você está navegando" and "para navegar", instead, we seem to be repeating the same concept of browsing. I had initially done the same mistake when translating to Italian 🙃. Would you mind taking a second look at this?